potpie/context-engine/adapters/inbound/cli/auth/gitlab_auth.py

```
@@ -0,0 +1,322 @@
+_GITLAB_OPEN_PROMPT_PREFIX = (
+    "Press Enter to open now, or browser opens in "
+)
+
+from adapters.outbound.cli_auth.gitlab_client import (
+    GitLabAuthErrorKind,
+    instance_host,
+    normalize_instance_url,
+    parse_user_profile,
+    verify_instance_access,
+    verify_read_api_scope,
+)
```

Ruff (0.15.20)

[error] Module level import not at top of file

(E402)